pkg/apis/kops/componentconfig.go

```diff
@@ -963,6 +963,10 @@ type SnapshotControllerConfig struct {
 
 // NodeTerminationHandlerSpec determines the node termination handler configuration.
 type NodeTerminationHandlerSpec struct {
+	// DeleteSqsMsgIfNodeNotFound makes node termination handler delete the SQS Message from the SQS Queue if the targeted node is not found.
+	// Only used in Queue Processor mode.
+	// Default: false
+	DeleteSqsMsgIfNodeNotFound *bool `json:"deleteSqsMsgIfNodeNotFound,omitempty"`
```

Comment:
SQS should be in uppercase.
```suggestion
	DeleteSQSMsgIfNodeNotFound *bool `json:"deleteSQSMsgIfNodeNotFound,omitempty"`
```

Reply:
Done.